src/Surfnet/StepupMiddleware/MiddlewareBundle/Tests/Service/SecondFactorDisplayNameResolverServiceTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Copyright 2020 SURF bv.
```

Comment:
:100: for adding the test!
Small nitpick, and feel free to leave as is, but the compyright notice should be: `Copyright 2020 SURF B.V.`